### ISSUE TITLE
Trying to reproduce for SemEval 2014

### ISSUE THREAD

**Issue by azamatolegen** [HSLCY/ABSA-BERT-pair#17]
Great article! Thank you for publishing the code!

I am following your steps and on step 3 when I run the code for semeval_NLI_M I got this error:

![image](https://user-images.githubusercontent.com/57138593/71049682-20e9a380-217e-11ea-8ed2-18ee23ebd2e0.png)
![image](https://user-images.githubusercontent.com/57138593/71049714-38c12780-217e-11ea-9728-44fb2e30407b.png)

could you please help to figure this out? I am using mBERT model.

**Comment by azamatolegen** [comment]
the code I am running is:

CUDA_VISIBLE_DEVICES=0,1,2,3 python run_classifier_TABSA.py --task_name semeval_NLI_M --data_dir data/semeval2014/bert-pair/ --vocab_file mBERT/vocab.txt --bert_config_file mBERT/config.json --init_checkpoint mBERT/pytorch_model.bin --eval_test --do_lower_case --max_seq_length 128 --train_batch_size 4 --learning_rate 5e-5 --num_train_epochs 2.0 --output_dir results/semeavl2014/NLI_M --seed 42

**Comment by sunny678** [comment]
Maybe you should check the differences between mBERT and BERT-base and modify the 'convert_tf_checkpoint_to_pytorch.py' file if necessary.

GPU memory related issues reference: https://github.com/HSLCY/ABSA-BERT-pair/issues/1
